Android.gitignore

```diff
@@ -1,85 +1,29 @@
-# Built application files
-*.apk
-*.aar
-*.ap_
-*.aab
-
-# Files for the ART/Dalvik VM
-*.dex
-
-# Java class files
-*.class
-
-# Generated files
-bin/
-gen/
-out/
-#  Uncomment the following line in case you need and you don't have the release build type files in your app
-# release/
-
 # Gradle files
 .gradle/
 build/
 
 # Local configuration file (sdk path, etc)
 local.properties
 
-# Proguard folder generated by Eclipse
-proguard/
-
-# Log Files
+# Log/OS Files
 *.log
+.DS_Store
```

Comment:
OS-specific files like this should already exist in a different template - in this case `Global/macOS.gitignore` - to avoid duplication. Can you drop this rule from the file?

Reply:
I completely agree with you that we should avoid unnecessary duplication. For this specific case, can I ask you to make an exception?

Removing this one line here would unavoidably result in **unintentionally committed** `.DS_Store` files in many repositories. A simple example would be: Someone not knowing about the `macOS.gitignore` file, just looking for something to get started on Android. Or someone else setting up a project on Linux, then people start contributing from macOS. `Global/macOS.gitignore` currently contains 26 lines (of which 25 would potentially be not applicable for Android).
Also, `.DS_Store` has been part of the default `.gitignore` generated by Android Studio for a long time, since many mobile developers use macOS.

If you really do think this line has to be removed for this PR to be merged, just leave a quick comment here, and I'll remove it immediately.

Reply:
Still existing?

Reply:
That's right, `.DS_Store` continues to be part of the default `.gitignore` as generated by Android Studio.

Reply:
Yeah, but here we are creating templates GH will generate from, so this still goes into global os specific gitignore or won't get merged..

Reply:
What are your thoughts on my comment above?